### PR TITLE
[Dawn] 0.6.0 Log Directory Fix

### PR DESCRIPTION
**Pull request by kevinmasd**
Due to Node and how __dirname is handled, this right now is the only clear way to handle pathing given current scheduling.

### CODE REVIEW

**Comment by kevinmasd** [comment]
Looking into that.

**Comment by kevinmasd** [comment]
Decided to shift to desktop location, one of the options definitely known to exist cross-platform. (The others are the temp folders or the download folder)

File: dawn/renderer/utils/utils.js [hunk 1]
```diff
@@ -81,7 +81,14 @@ export const windowInfo = {
 
 export class Logger {
   constructor(processname, firstline) {
-    this.log_file = fs.createWriteStream(`./${Date.now()}-${processname}.log`, { flags: 'w' });
+    let path;
+    if (processname === 'dawn') {
+      path = require('electron').remote.app.getPath('desktop'); // eslint-disable-line global-require
```
Comment (kevinji) [inline dawn/renderer/utils/utils.js:86]: Question: Is there a reason this isn't imported at the top of the file?

Comment (kevinmasd) [reply dawn/renderer/utils/utils.js:86]: Main vs renderer difference. One throws an error if used in one process, the other throws an error if used in the other process.

**Review by kevinji — APPROVED** [review]
Looks okay to me. I think #314 should also be merged in after this to use append mode instead of write mode for log files.

**Comment by kevinmasd** [comment]
It's already released with this feature. Any other changes will not be in 0.6.0

**Pull request merged**